server/src/main/java/com/coincoinche/websockets/controllers/GameController.java

```diff
@@ -0,0 +1,50 @@
+package com.coincoinche.websockets.controllers;
+
+import com.coincoinche.engine.CoincheGame;
+import com.coincoinche.events.RoundStartedEvent;
+import com.coincoinche.store.GameStore;
+import com.coincoinche.store.InMemoryGameStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class GameController {
+  private static final Logger logger = LoggerFactory.getLogger(GameController.class);
+
+  @Autowired private SimpMessagingTemplate template;
+  private GameStore store;
+
+  public GameController() {
+    this.store = new InMemoryGameStore();
+  }
+
+  /**
+   * To be called after the client loaded the game. Send its hand to the player.
+   *
+   * @param gameId - id of the game
+   * @param username - username of the player
+   */
+  @MessageMapping("/game/{gameId}/player/{username}/ready")
```

Comment:
Once 4 players have joined, the following happens:
- The lobby (for now) inits the game, 
- The lobby (for now) registers the new game with `registerNewGame` in this controller.
- The lobby sends the gameId to the 4 clients
- Each client calls /game/{gameId}/player/{username}/ready with the gameId and its username.
- The GameController sends back the first event of the game: a RoundStartedEvent containing the cards for the new round. The topic used for each player is /topic/game/{gameId}/player/{username}, it will be the same for all the other game events.
- The front end can display the right cards.

With this, we now support several games at once (even without multithreading, to be added later) 🎉

Reply:
Great! We can keep this for now, but in the future it may be better to use authentication for a player to join a game instead of having the username in the URI.

Reply:
Yes clearly, for now there is no auth whatsoever you can see the cards of the other players by subscribing to their topics :p